### ISSUE TITLE
Nested menus vs. required owned elements

### ISSUE THREAD

**Issue by smhigley** [w3c/aria#1425]
This is related to #1033 in that it may or may not be an issue, depending on the interpretation of "required owned elements" :D.

I'd just like to suggest a clarification on the menu role, specifically around nested dropdown menus. Right now, the structure of required owned elements / required context role suggests the following structure:

```
menu / menubar > group (optional) > menuitem / -checkbox / -radio
```

The problem is that often nested dropdown  menus end up creating a role structure where `menu` is a child of a parent `menu` or `menubar` (or sometimes even placed under `menuitem`). Even if this is technically allowed under a permissive reading of "required owned elements", I think it'd be useful to clarify where the nested `menu` should go.

A practical example of this is in the APG menu examples, where the dropdown `menu` is a child of `menubar`: https://w3c.github.io/aria-practices/examples/menubar/menubar-editor.html

**Comment by patrickhlauke** [comment]
coming in late, but as this just bubbled up in a discussion: yes, it would be good to clarify (even if it's just in prose) what the story with nested menus is. an initial skim over the spec seems to suggest that a `role="menu"` can't be a direct child of another `role="menu"`...

**Comment by Z3rp** [comment]
Yes, I was discussing with Patrick and Adrian on the a11y slack channel about this issue. To me it seems that the wording of the 5.2.6 Allowed Accessibility Child Roles does not allow a `role=menu` to be nested inside another `role=menu` given the wording 

> A list of roles which are allowed on an accessibility child (simplified as "child") of the element with this role. Authors MUST only add child element with allowed roles.

**Comment by flavi1** [comment]
Does this means that nested menus have to be achieved with the group role to declare submenus? Is it sementically ok to use group role instead of menu role for submenus?

**Comment by JAWS-test** [comment]
> Is it sementically ok to use group role instead of menu role for submenus?

No. If I understand it correctly, submenus are implemented with role=menu (nested in the parent menu). role=group is not for submenu, but for a grouping within a menu (see https://www.w3.org/WAI/ARIA/apg/patterns/menubar/examples/menubar-editor/ - submenu "Size" uses role=group for "X-Small" to "X-Large")

**Comment by dd8** [comment]
There's another example of nested `menu` roles in
https://www.w3.org/WAI/ARIA/apg/patterns/menubar/examples/menubar-navigation/ 

There are no Required Accessibility Parent Roles for `menu` so looks like nested sub-menus can be handled by adding `menu` to the list of Allowed Accessibility Child Roles for the `menu` role.

I can submit a PR for this
